bindings/2.206/GeometryDash.bro

```diff
@@ -6174,7 +6174,32 @@ class GJAccountSettingsDelegate {
 class GJAccountSettingsLayer : FLAlertLayer, TextInputDelegate {
 	// virtual ~GJAccountSettingsLayer();
 
-	static GJAccountSettingsLayer* create(int);
+        inline GJAccountSettingsLayer() {
+        m_accountID = 0;
+        m_messageStatus = 0;
+        m_friendStatus = 0;
+        m_commentHistoryStatus = 0;
+        m_youtubeURL = "";
+        m_twitterURL = "";
+        m_twitchURL = "";
+        m_youtubeInput = nullptr;
+        m_twitterInput = nullptr;
+        m_twitchInput = nullptr;
+        m_messageSettings = nullptr;
+        m_friendRequestSettings = nullptr;
+        m_commentSettings = nullptr;
+       }
+
+	static GJAccountSettingsLayer* create(int a1)
+        {
+        GJAccountSettingsLayer* pRet = new GJAccountSettingsLayer();
+        if (pRet && pRet->init(a1)) {
+        pRet->autorelease();
+        return pRet;
+        } 
+        CC_SAFE_DELETE(pRet);
+        return nullptr;
+        }
```

Comment:
can you at least try with the indentation :(

Reply:
my bad, i just paste the code of my bindings 😭